src/web/landing/templates/base.html.twig

```diff
@@ -1,3 +1,4 @@
+<!doctype html>
```

Comment:
To prevent using compatibility mode in browser: https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode